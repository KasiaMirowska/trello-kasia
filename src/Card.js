import React from 'react';

export default function List(props) {
    console.log(props)
    //    const card = props.cards.map(id => {
    //     <Card key = id/> })
    return (
        <div className="App-list">
            <section className="List">
                <header className="List-header">
                    <h2>{props.header}</h2>
                </header>
                <div className="List-cards">
                    <div className="Card">
                        <button type="button">delete</button>
                        <h3>{props.title}</h3>
                        <p>{props.content}</p>
                    </div>

                </div>
            </section>
        </div>
    )
}
