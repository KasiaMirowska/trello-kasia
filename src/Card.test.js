import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import './Card.css';
import Card from './Card';


it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card title = 'A title' content = 'some content'/>, div);
    ReactDOM.unmountComponentAtNode(div);
})

it('renders as expected', () => {
    const tree = renderer.create(<Card title='A title' content='some content' />)
    expect(tree).toMatchSnapshot();
})