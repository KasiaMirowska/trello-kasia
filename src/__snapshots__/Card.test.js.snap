// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    A title
  </h3>
  <p>
    some content
  </p>
</div>
`;
